uiautomationutilities/perf_tool/MathUtils.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace PerfClTool.Measurement
+{
+    internal class MathUtils
+    {
+        public static double GetStdDeviation(List<double> values)
+        {
+            var average = values.Average();
+            if (values.Count == 1)
+            {
+                return 0.0;
+            }
+
+            var stdev = 0.0;
+
+            for (var n = 0; n < values.Count; n++)
+            {
+                stdev += Math.Pow(values[n] - average, 2);
+            }
+
+            stdev = Math.Sqrt(stdev / (values.Count - 1));
+
+            return stdev;
+        }
+        public static double GetPercentile(List<double> values, int percentile)
```

Comment:
How many values are we expecting in this list?

Reply:
@AdamBJohnsonx , the number of times the test is running (Number of iterations of same scenario), will be the length of the list.